packages/documentation/src/stories/internet-header/components/footer/custom-config/footer-config.docs.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import * as FooterConfig from './footer-config.stories';
+
+<Meta of={FooterConfig} />
+
+# Footer with custom config
+
+<div className="lead">When you need to add your own footer links.</div>
+
+<Canvas of={FooterConfig.Default} />
+
+<div className="container">
+  <Controls of={FooterConfig.Default} />
+</div>
```

Comment:
```suggestion
<Controls of={FooterConfig.Default} />
```

Reply:
This one is necessary as the layout was transformed to get maximum width for the component canvas.

Reply:
In this case, the Heading should be wrapper in a `.container` as well:
![image](https://github.com/swisspost/design-system/assets/9716662/7e4205c7-c079-4455-9b66-dc33edaa93a9)